xl_auth/user/models.py

```diff
@@ -57,6 +59,48 @@ def get_by_email_and_code(email, code):
         if user:
             return PasswordReset.query.filter_by(code=code, user=user).first()
 
+    def send_email(self):
+        """Email password reset link to the user."""
+        password_reset_url = url_for('public.reset_password', email=self.user.email,
+                                     code=self.code, _external=True)
+        service_name = current_app.config['SERVER_NAME'] or current_app.config['APP_NAME']
```

Comment:
@anjabergman, @Hardesjo: Do you have any thoughts regarding what we should put in the "From" field in a password reset email? Right now it would be something like "login.libris.kb.se <noreply@kb.se>". Other possible contenders are "Libris Login <noreply@kb.se>" and "noreply@kb.se". Also, should we use the same in the "Subject" field?

Reply:
@anjabergman @Hardesjo Rent konkret ser det ut såhär idag:

<img width="640" alt="screen shot 2017-11-15 at 9 40 59 am" src="https://user-images.githubusercontent.com/786326/32826569-336d5658-c9e9-11e7-93f1-5cf883a9620d.png">

Så, frågan ifall något ska ändras i subject eller avsändare eller annan ställe?